op-devstack/sysext/helpers.go

```diff
@@ -18,11 +19,13 @@ const (
 	RBuilderServiceName  = "rbuilder"
 	ConductorServiceName = "conductor"
 
-	HTTPProtocol    = "http"
-	RPCProtocol     = "rpc"
-	MetricsProtocol = "metrics"
+	HTTPProtocol                 = "http"
+	RPCProtocol                  = "rpc"
+	MetricsProtocol              = "metrics"
+	WebsocketFlashblocksProtocol = "ws-flashblocks"
 
-	FeatureInterop = "interop"
+	FeatureInterop     = "interop"
+	FeatureFlashblocks = "flashblocks"
```

Comment:
Where is this used?

Reply:
Ahh forgot to remove it with a function I removed relying on this var.